vc_assistance/indico_vc_assistance/util.py

```diff
@@ -0,0 +1,60 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from sqlalchemy.orm import joinedload
+
+from indico.core.db.sqlalchemy.util.queries import limit_groups
+from indico.modules.events import Event
+from indico.modules.events.requests.models.requests import Request, RequestState
+from indico.modules.vc import VCRoomEventAssociation
+
+
+def is_vc_manager(user):
```

Comment:
Very misleading name. This is not a list of "managers" but a list of people who can request assistance, so what about something like `can_request_assistance` for the function and `authorized` for the acl setting?